mlflow/tensorflow/__init__.py

```diff
@@ -194,6 +207,27 @@ def log_model(
     :return: A :py:class:`ModelInfo <mlflow.models.model.ModelInfo>` instance that contains the
              metadata of the logged model.
     """
+    if keras_model is not None:
+        if tf_saved_model_dir is not None or tf_meta_graph_tags is not None or tf_signature_def_key is not None:
```

Comment:
```suggestion
    if keras_model is not None and tf_saved_model_dir is not None or tf_meta_graph_tags is not None or tf_signature_def_key is not None:
```

Reply:
But there's a "return" command in `if keras_model is not None:` scope

Reply:
An got it.